compiler/rustc_metadata/src/dependency_format.rs

```diff
@@ -397,18 +401,14 @@ fn verify_ok(tcx: TyCtxt<'_>, list: &[Linkage]) {
             if let Linkage::NotLinked = *linkage {
                 continue;
             }
-            if desired_strategy == PanicStrategy::Abort {
-                continue;
-            }
```

Comment:
This is a key part of the PR, right? That's where previously we skipped a check that we do not skip any more now.

Reply:
Sorts of. Previously any crate can be linked to panic-abort; this line only has to be removed because the required panic strategy becomes three-value. You can think of the all previous `PanicStrategy::Unwind` maps to `None` in this PR.